src/pages/data-science-tools/jupyter-notebook/index.md

```diff
@@ -10,4 +10,6 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
 #### More Information:
-[Jupyter/IPython Notebook Quick Start Guide](http://jupyter-notebook-beginner-guide.readthedocs.io/en/latest/what_is_jupyter.html)
+<!-- Please add any articles you think might be helpful to read before writing the article -->
```

Comment:
Hmmm, looks like there was only a link added to this stub, that the Normalise script removes. Should we add it back, or will this keep happening until the stub language is replaced with an article?